src/Examine.Test/Examine.Test.csproj

```diff
@@ -11,7 +11,7 @@
     </SccProvider>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Can we multi-target this so there are no breaking changes?

Reply:
I've now added multi targeting to the test project. This means that the tests will run on both .NET 5 and .NET 6 when running `dotnet test`. Do note that the Visual Studio test explorer only runs the tests on one framework (The one placed first). Source: https://stackoverflow.com/questions/44477840/how-to-properly-unit-test-a-net-project-with-multiple-target-frameworks-given

I've also added .NET 6 support to the build workflow to allow the GitHub action to run the tests in .NET 6.